apple/MarkdownCommitHook.mm

```diff
@@ -170,8 +170,8 @@
             }
 
             // apply markdown
-            auto newString = [utils parseMarkdown:nsAttributedString
-                                   withAttributes:defaultNSTextAttributes];
+            NSMutableAttributedString *newString = [nsAttributedString mutableCopy];
```

Comment:
Wouldn't it be better to use `auto` here as before? And the same for other variables in this whole PR. I can mark them all if you wish

Reply:
This is done kind of on purpose, in `react-native` codebase there's no single occurrence of `"auto "` in `*.m` files (there are 478 occurrences in `*.mm` but they are rather in C++ code). I think I'd like to stick to the same codestyle as in `react-native` for now.